Dev

nparra75 opened the pull request.

afislonge committed:
Create imagegetty.ipynb file

afislonge committed:
get deepfake image

afislonge committed:
Create README.md

create UI project folder

afislonge committed:
Create README.md

Model project folder

afislonge committed:
Create README.md

fake and real image code folder

afislonge committed:
Create README.md

project utilities code folder for creating mongo db, moving images to project google drive

Commit:
Created the initial version of the UI; code integration with the model is included, but in the meantime, it is commented.

nparra75 committed:
MongoDB Creation and Collection

nparra75 committed:
It was agregated an environment var to connection string

nparra75 committed:
modified README.md file

isamia-can-code committed:
Update README.md

Updating README.md to reflect requirements for running moveImages.py

isamia-can-code committed:
Adding moveImages.py to utilities folder

moveImages.py contains code to move images from a specific drive folder to another.

GanzB02 committed:
Create FacialDetection_Landmarks.py

Facial Detection and landmarks feature Extraction from thispersondoesnotexist.com

GanzB02 committed:
Update FacialDetection_Landmarks.py

Added .dat file path for Shape_Predictor_Dlib

meenu16411 committed:
Initial code for CNN Model

meenu16411 committed:
Deleted MapReduce_part_1.ipynb

Commit:
First version of the VGG19 model, that applies transfer learning from the VGG19 model

Mikeltec committed:
Python code to resize images from Google Drive /RealData and /FakeImages, using Google API.

afislonge committed:
optmise the code, add processing count

Commit:
This is an update of the VGG19 model, with the training of 4000 images as the sample data

isamia-can-code committed:
Code for Xception_Model3

Model3 was generated after 3 different versions of this code and multiple hyperparamater tuning.

GanzB02 committed:
Add files via upload

Contains Feature Engineering Pipeline 2 : Feature Extraction and TSNE

GanzB02 committed:
Rename Mid_Submission___Notebook_ganb (1).ipynb to Feature_Engineering_TSNE

GanzB02 committed:
Delete imagepreprocessor/Feature_Engineering_TSNE

GanzB02 committed:
Add files via upload

Feature Engineering Pipeline 2
Steps defined in ipynb

meenu16411 committed:
Jupyter Notebook for CNN Model with 4000 Images

GanzB02 committed:
Update README.md

GanzB02 committed:
Add files via upload

Feature Engg Pipeline 2: Advanced Analytics using FeatUP

GanzB02 committed:
Rename Copy_of_Feature_Engg_2_Advanced_Image_Analytics.ipynb to Feature_Engg_2_Advanced_Image_Analytics_ganb.ipynb

GanzB02 committed:
Delete imagepreprocessor/Feature_Engg_2_Advanced_Image_Analytics_ganb.ipynb

GanzB02 committed:
Add files via upload

Feature Engg Pipeline 2: Advance Image Detection

nparra75 committed:
Script yo create flask app

afislonge committed:
style the project UX

afislonge committed:
Merge branch 'dev' of https://github.com/afislonge/deepfake-detection-project into dev

afislonge committed:
style the project UX

nparra75 committed:
Create .gitkeep

code related to deploy

nparra75 committed:
Moved app.py file to new directory

nparra75 committed:
Moved app.py file to new directory

Commit:
Code implementing LIME to verify the classification

afislonge committed:
aligned and bold the report text

nparra75 committed:
new API to run from colab

nparra75 committed:
Added Xception model load of file model_epoch_10_val_accuracy_1.0000.h5

GanzB02 committed:
aligned and bold the report text (#44)

Reconstruct the UI look and feel. 

![image](https://github.com/afislonge/deepfake-detection-project/assets/19928943/b677ed21-5d3d-4753-9176-58aa6b416d44)

nparra75 committed:
Update Flask API to load Xception model saved for internal test

nparra75 committed:
Added request POST to API URL

JuanS286 committed:
Final update of VGG19 model and file in py format

isamia-can-code committed:
Add files via upload

Script to figure out which pictures are in Test, Train and Validation data

Mikeltec commented on deploy/templates/index.html:
Please remove this file.

nparra75 committed:
Added load of VGG and Xception models

nparra75 committed:
Added VGG and Xception models

nparra75 committed:
Create folders of models VGG19

nparra75 committed:
Create folder of Xception model

Mikeltec committed:
New logo integration.

Mikeltec committed:
New logo integration.

Mikeltec committed:
Merge branch 'dev' of https://github.com/afislonge/deepfake-detection-project into dev

Mikeltec committed:
New logo integration.

meenu16411 committed:
Face detection

meenu16411 committed:
Merge branch 'dev' of https://github.com/afislonge/deepfake-detection-project into dev

Mikeltec committed:
Deep learning models.

meenu16411 committed:
Face detection by uploading images

isamia-can-code committed:
Replacing First Xception Model Training Finetuning

Replacing the First Xception Model Training & Finetuning - added different hyperparameters tuning and confusion matrix

isamia-can-code committed:
LIME interpretation & model training 2

afislonge committed:
added facialrecognition and readme file

JuanS286 committed:
Final version of UI that was deployed

afislonge committed:
modified readme content

meenu16411 committed:
Face Detection Deployment on Streamlit

nparra75 committed:
added the call to the face recognition model

nparra75 committed:
Merge branch 'dev' of https://github.com/afislonge/deepfake-detection-project into dev

nparra75 committed:
Added libs necessary

JuanS286 committed:
Little change in the code to make it works for deployment
